source/renderer/app/components/wallet/WalletCreateDialog.js

```diff
@@ -211,9 +211,23 @@ export default class WalletCreateDialog extends Component<Props, State> {
     const { isSubmitting } = this.state;
     const dialogClasses = classnames([styles.component, 'WalletCreateDialog']);
 
+    const walletNameField = form.$('walletName');
+    const spendingPasswordField = form.$('spendingPassword');
+    const repeatedPasswordField = form.$('repeatPassword');
+
+    const canSubmit =
```

Comment:
@DeeJayElly this is working but is very ugly. Can you try to use form validation instead of checking all individual fields? Please check the documentation here: https://foxhound87.github.io/mobx-react-form/docs/form/

Reply:
Added